examples/vision/semisupervised_simclr.py

```diff
@@ -197,14 +207,14 @@ def call(self, images, training=True):
 
 # Image augmentation module
 def get_augmenter(min_area, brightness, jitter):
-    zoom_factor = 1.0 - tf.sqrt(min_area)
+    zoom_factor = 1.0 - np.sqrt(min_area)

```

Comment:
It really is `tf`

Reply:
I forgot to mention the reason I changed that. When using _tf.sqrt()_ and trying to save the model, there is a JSON serialization error:
`Unable to serialize 0.0669873058795929 to JSON. Unrecognized type <class 'tensorflow.python.framework.ops.EagerTensor'>.` 

Using _np.sqrt()_ instead was my workaround. I suppose there might be a better solution for this but my tried approaches did not solve the issue.

Reply:
Got it -- in that case consider using `math.sqrt` from the Python stdlib.